### DISCUSSION TITLE
没有权限 您尚未授予必要的权限，请在设置中授予权限

### DISCUSSION THREAD

**Discussion by garsiazero** [localsend/localsend#1668] [L-简体中文]
![image](https://github.com/user-attachments/assets/72c4cbd1-7038-47d4-8908-c6cdccec6454)
旧盘快报废了，想着赶紧拯救一下，用这个局域网传输软件，结果来这么一下

是用的共享权限还是安全权限？安全权限已经默认是完全控制了，旧盘已经快报废了，再设置共享权限会死的

**Comment by lewissss9595** [comment]
解决了吗

**Reply by MTcool** [reply]
试试这个方法 https://github.com/localsend/localsend/discussions/835
